environment_osx.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Could you please also mention this file in [the developer documentation on updating dependencies](https://docs.esmvaltool.org/en/latest/community/code_documentation.html#dependencies)?